e2e/tests/localAuthorityMaintainsCaseAfterGatekeeping_test.js

```diff
@@ -36,6 +36,21 @@ Before(async (I, caseViewPage, submitApplicationEventPage, enterFamilyManCaseNum
   await I.navigateToCaseDetails(caseId);
 });
 
-Scenario('local authority uploads documents', uploadDocs.uploadDocuments());
+Scenario('local authority uploads documents', async (I, caseViewPage, uploadDocumentsEventPage) => {
+  await caseViewPage.goToNewActions(config.applicationActions.uploadDocuments);
+  uploadDocs.uploadMandatoryDocuments(uploadDocumentsEventPage);
+  await I.completeEvent('Save and continue');
+  I.seeEventSubmissionConfirmation(config.applicationActions.uploadDocuments);
+  caseViewPage.selectTab(caseViewPage.tabs.documents);
+  uploadDocs.assertMandatoryDocuments();
+});
 
-Scenario('local authority uploads court bundle', uploadDocs.uploadCourtBundle());
+Scenario('local authority uploads court bundle', async (I, caseViewPage, uploadDocumentsEventPage) => {
+  await caseViewPage.goToNewActions(config.applicationActions.uploadDocuments);
+  I.seeElement(uploadDocumentsEventPage.documents.courtBundle);
```

Comment:
maybe it should be part of uploadDocumentsEventPage.uploadCourtBundle

Reply:
Page method should just upload document in my opinion.